thetis/coupled_timeintegrator.py

```diff
@@ -493,8 +491,8 @@ def __init__(self, solver):
             explicit_v_diff = solver.tot_v_diff.get_sum()
 
         vdiff_sp = {'snes_monitor': False,
-                    'ksp_type': 'gmres',
-                    'pc_type': 'ilu',
+                    # 'ksp_type': 'gmres',
+                    # 'pc_type': 'ilu',
```

Comment:
We could revisit these bits next week, but if you want ilu but safe in parallel use `'pc_type': 'bjacobi', 'sub_pc_type': 'ilu'`.  Which does process-block jacobi and ilu on the blocks.